### DISCUSSION TITLE
Use `README.md` / `CONTRIBUTING.md` / `CODE_OF_CONDUCT.md`

### DISCUSSION THREAD

**Discussion by vinitshahdeo** [CatsInTech/GitHub-ReadMe#37] [General]
Let's use `README.md` / `CONTRIBUTING.md` / `CODE_OF_CONDUCT.md` in GitHub repositories.

- On GitHub, the commonly used and recognized convention for the filename of the README file is "**README.md**". This is case-sensitive, so it should be written in uppercase for "README" and lowercase for ".md" (Markdown file extension). Using "README.md" as the filename ensures compatibility and consistency across different repositories and platforms. It is also the default filename that GitHub recognizes and displays prominently on the repository's main page. While some platforms may accept alternative filename variations like "Readme.md" or "readme.md", it is generally recommended to follow the "README.md" convention for better consistency and compatibility within the GitHub ecosystem.

- On GitHub, the convention for the filename of the contributing guidelines file is typically "**CONTRIBUTING.md**". This convention is widely recognized and followed by many open-source projects and repositories. By using "CONTRIBUTING.md" as the filename, you ensure consistency and compatibility with other repositories and projects on GitHub. This naming convention is also recognized by various tools and integrations that rely on standard file naming conventions for contributing guidelines.

- GitHub recognizes "**CODE_OF_CONDUCT.md**" as the default filename for the Code of Conduct file. Using this convention ensures consistency and compatibility with other repositories and projects on the platform. It also aligns with the expectations of various tools and integrations that rely on standard file naming conventions for Code of Conduct files.

@praveenscience wdyt?

**Comment by praveenscience** [comment]
Yea, absolutely... I can do it myself, but will let other folks take it up. Created an issue for the same! 😊
